src/server/middleware/sendHtml.js

```diff
@@ -40,8 +40,8 @@ function getChunkAssets(assetsByChunkName) {
     .map(([, assets]) => (typeof assets === 'string' ? assets : assets[0]));
 }
 
-const modernBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.webpack-stats.browser.json').assetsByChunkName);
-const legacyBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.webpack-stats.legacyBrowser.json').assetsByChunkName)
+const modernBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.build-meta.json').modernBrowserChunkAssets);
+const legacyBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.build-meta.json').legacyBrowserChunkAssets)
```

Comment:
If you don't go for [destructuring](https://github.com/americanexpress/one-app/pull/22/files#r373077172), be sure to include the semicolon. I'm a bit confused on why linting didn't pick this up, though?
```suggestion
const legacyBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.build-meta.json').legacyBrowserChunkAssets);
```

Reply:
I'll look into why linting didnt pick this up. That is weird.

Reply:
The reason as to the semicolon is omitted is bc that line in question is part of:

```js
const legacyBrowserChunkAssets = getChunkAssets(readJsonFile('../../../.build-meta.json').legacyBrowserChunkAssets)
  .map((chunkAsset) => `legacy/${chunkAsset}`);
```

Reply:
And the reason for no destructuring is that `readJsonFile` is what is returning the object that includes the chunk assets. See the parentheses.

Reply:
might save memory and CPU if the file is read and parsed only once, IDK when the garbage collection would run after the first parse so two copies might still be in memory at that point